content/2021-12-01-drogue-cloud-zero-eight/index.md

```diff
@@ -0,0 +1,142 @@
+effort into cleaning up, filling some gaps that we identified, and polish the current functionality.
+
+<!-- more -->
+
+## Improved web console
+
+We believe that the web console plays an important role into getting started with Drogue Cloud. Sure, once your know
+your way around, a CLI like `drg` or directly using our REST API might better for you. But until that point, the
+web console should allow you the get the basics done in a manual way.
+
+That is why we expanded the current functionality of the web console with features that already had been available
+under the hood. Adding users to an application, transferring ownership, or directly monitoring traffic from the
+application view for example.
+
+![Console screenshot](console-1.png)
+
+As always, we will also update our public sandbox with the most recent, released version. So you can directly try out
+the new features.
+
+## Kafka topic provisioning, reloaded!
+
+In 0.7 we introduced Kafka topic provisioning, using Strimzi. Creating a new application will automatically spin up
+a new Kafka topic for it, and setup access. We already designed this as an extension point, so that one could drop
+in an alternative implementation of that.
+
+Well, in 0.8 we did that. In addition to the Strimzi based topic provisioner, we added a plain Kafka Admin client
+based one. And while the downside is, that this cannot create a new user for the consuming side for you, it has the
+benefit that it works with any Kafka service that provides the Kafka Admin API.
+
+We will follow up with an additional blog post on this the near future. So, stay tuned if you are interested!
+
+## Improved gateway device support
+
+During the authentication and authorization process of devices, we already supported the idea of a gateway. However,
+once the device was connected, we didn't. For session-less protocols (like HTTP or CoAP) this didn't really matter.
+For MQTT however, it does. MQTT allows a device to connect (and authenticate). If however that device is a
+gateway, it should be able to publish on behalf of other devices, without the need to reconnect.
+
+So for Drogue Cloud 0.8, we added the ability for MQTT to authorize publish (events) and subscribe (commands) operations
+independently of the MQTT connection phase. Meaning: MQTT gateways can now act on behalf of other devices, if they are
+configured to allow this. As the concept existed before 0.8, no new configuration had to be added, just the APIs for the
+MQTT endpoint got extended to support this.
+
+## Static server binary
+
+Deploying a complex, cloud native stack like Drogue Cloud can be tricky. We put a lot of effort in our installer scripts
+and Helm charts to make it both easy and customizable. However, every now and then, it would be nice to "just run"
+Drogue Cloud. With 0.8, you can!
+
+Assuming you have an existing Kafka and Keycloak instance, it is as easy as:
```

Comment:
And Postgres instance!

Reply:
It's worth mentionning that getting the aforementioned postgres, keykloack and kafka can be done as easy as `docker compose` as we provide the file.